yscope-compose/coordinator/scripts/generate-configs.sh

```
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+# Exit on error
+set -e
+
+PRESTO_CONFIG_DIR="/opt/presto-server/etc"
+
+# Substitute environemnt variables in config template
+find /configs -type f | while read -r f; do
+  ( echo "cat <<EOF"; cat $f; echo "EOF" ) | sh > "${PRESTO_CONFIG_DIR}/$(basename "$f")"
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)